sdk/metric/internal/x/x.go

```diff
@@ -67,3 +67,14 @@ func (f Feature[T]) Enabled() bool {
 	_, ok := f.Lookup()
 	return ok
 }
+
+// EnabledInstrument provides the interface for synchronous
+// instruments to returned whether or not they are enabled.
+type EnabledInstrument interface {
+	// Enabled returns whether the instrument has any measurements
+	// associated with it.
```

Comment:
```suggestion
	// Enabled returns whether the instrument has any measures
	// associated with it.
```

However, I would prefer

```suggestion
	// Enabled returns whether the instrument will process measurements for the given context.
```